pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -302,6 +304,25 @@ func NewManager() (*LocalGadgetManager, error) {
 		return nil, err
 	}
 
+	// Ensure /sys/fs/bpf is of type bpf. It is necessary to be able to pin eBPF
+	// maps. TODO: Remove the need of using pinning, see issues #619 and #620.
+	cmd := exec.Command("/bin/sh", "-c", "stat -f --format=%T /sys/fs/bpf")
+	output, err := cmd.Output()
+	if err != nil {
+		return nil, fmt.Errorf("error checking type of /sys/fs/bpf (%s): %w",
+			cmd.String(), err)
+	}
+	fsType := strings.TrimSpace(string(output))
+	if fsType != "bpf_fs" {
+		log.Infof("/sys/fs/bpf is of type %s. Remounting with type bpf", fsType)
+
+		cmd := exec.Command("/bin/sh", "-c", "mount -t bpf bpf /sys/fs/bpf/")
```

Comment:
Could use https://pkg.go.dev/golang.org/x/sys/unix#Mount

[Example of usage](https://github.com/opencontainers/runc/blob/238717ddd31cff54b36a6ae937c4016d501caaa8/libcontainer/mount_linux.go#L57)